.mergify.yml

```diff
@@ -0,0 +1,29 @@
+pull_request_rules:
+  - name: assign and label scala-steward's PRs
+    conditions:
+      - author=scala-steward
+    actions:
+      assign:
+        users: [jendakol, sideeffffect]
+      label:
+        add: [dependency-update]
+  - name: merge Scala Steward's PRs
+    conditions:
+      - base=master
+      - author=scala-steward
+      - status-success=continuous-integration/travis-ci/pr
+    actions:
+      merge:
+        method: squash
+  - name: automatic merge on CI success and review
+    conditions:
+      - base=master
+      - "#review-requested=0"
+      - "#changes-requested-reviews-by=0"
+      - "#approved-reviews-by>=1"
+      - status-success=continuous-integration/travis-ci/pr
+    actions:
+      merge:
+        method: squash
```

Comment:
So this will actually squash every PR, even some "normal" one? I disagree, IMHO it should be a choice of the author what to do with it. Does it even make sense to merge such PR automatically? What if the author wants to wait because he realized sth is missing, even though he has green both from CI and reviewers? I am not sure at all this is a good idea.  
I'd probably stick with automatic PRs merging.

Reply:
GitHub.com now allows us to mark a PR as `Draft` so I suppose that Mergify will not merge such PR.

Reply:
It seems that Mergify actually doesn't ignore drafts by default! So I would prefer if we could add `"-draft"` to the conditions.
https://doc.mergify.io/conditions.html

Reply:
Still there's the problem with squashing. I got used to using fixup commits and manual squashing before merging and I want to keep it however this will merge the PR before I'll have a chance to do that. BTW the main difference is that sometimes you just want to merge two commits from branch because it just makes sense.

Reply:
Looks like we need also `"-author=jendakol"` condition 😄